prt/tests/compute/prt_compute.lua

```diff
@@ -18,32 +18,65 @@ local blockchain_utils = require "blockchain.utils"
 local time = require "utils.time"
 local blockchain_constants = require "blockchain.constants"
 local Blockchain = require "blockchain.node"
+local CommitmentBuilder = require "computation.commitment"
+
+local function write_json_file(leafs, root_tournament)
+    local prev_accumulated = 0
+    local new_leafs = {}
+    for i, leaf in ipairs(leafs) do
+        new_leafs[i] = {
+            hash = leaf.hash,
+            repetitions = leaf.accumulated_count - prev_accumulated
+        }
+        prev_accumulated = leaf.accumulated_count
+    end
+    local inputs_and_leafs = {
+        leafs = new_leafs
+    }
+
+    local flat = require "utils.flat"
+    local json = require "utils.json"
+    local file_path = string.format("/dispute_data/%s/inputs_and_leafs.json", root_tournament)
+    local file = io.open(file_path, "w")
+
+    if file then
+        file:write(json.encode(flat.flatten(inputs_and_leafs).flat_object))
+        file:close()
+    end
```

Comment:
We should ensure that `io.open` and `file:close` succeed.

```suggestion
    local file = assert(io.open(file_path, "w"))
    file:write(json.encode(flat.flatten(inputs_and_leafs).flat_object))
    assert(file:close())
```